src/awesomeSezzle.js

```diff
@@ -820,6 +822,55 @@ class AwesomeSezzle {
       modalNode.style.display = 'none';
       modalNode.tabindex='-1';
       modalNode.role = 'dialog';
+      if(this.theme==="grayscale")
+      {
+        modalNode.innerHTML = `<div class="sezzle-checkout-modal-hidden1"> <div class="sezzle-modal1" title=""> <div class="sezzle-modal-content1"> <div class="sezzle-logo1" title="Sezzle logo"></div><button title="Close Sezzle Modal" class="close-sezzle-modal1" tabindex="0" role="button"></button>
```

Comment:
Would be nice to move all this html code to a different file to avoid cluttering

Reply:
I agree, but I also think we can do this in a smarter way by inserting classes for grayscale vs color to change only the necessary values.

Also, I don't think this should be tied to `theme` but be a separate setting, since grayscale might be selected for contrasting against a bright color background but the merchant might still want the color modal.